docs/en/features/local_census.md

```diff
@@ -1,33 +1,36 @@
+This feature is designed for installations with limited resources that cannot configure and customize their census connection remotely, so we offer them a solution to use a local census instead. For this purpose, administrators are provided with a way to manage the local census database through the administration panel under **Settings > Local census**.
 
-Allow administrators users to manage this table in two different ways:
+## Manage local census
 
-- **Manually**: one by one through a CRUD interface.
-- **Automatically**: through an importation process.
+Administrators can manage this census in two different ways:
 
-## Manually
+* **Manually**: add one by one through a form.
+* **Automatically**: through an importation process.
 
-Provide a way to manage local census records to administrator users through administration interface.
+### Manually
 
-- Local Census Page
-  ![Manage local census](../../img/local_census/manage-local-census-en.png)
+It allows administrators to create a record by clicking the _Create new local census record_ button, which appears in the top right corner, as shown in the following image:
 
-- Add new record
-  ![Create local census record](../../img/local_census/add-local-census-record-en.png)
+![Create a record in the local census button](../../img/local_census/manage-local-census-en.png)
 
-Features:
+This will take us to a new page where we can fill in the following form and create the new record:
 
-1. Search by document_number: As local_census_records could contain a lot of records we have added a search feature to allow administrators to find existing records by document_number.
-1. Avoid the introduction of duplicated records: A model validation has been added to the following attributes pair [:document_number, :document_type]
+![Form to create a record](../../img/local_census/add-local-census-record-en.png)
 
-## Automatically
+### Automatically
 
-Allow administrator users to import local census records though CSV file.
+It allows administrator to import multiple records through a CSV file by clicking the _Import CSV_ button, which appears in the top right corner, as shown in the following image:
 
-- Local Census Page
-  ![Manage local census csv](../../img/local_census/manage-local-census-csv-en.png)
+![Import csv button](../../img/local_census/manage-local-census-csv-en.png)
 
-- Import CSV
-  ![Create local census records csv](../../img/local_census/add-local-census-records-csv-en.png)
+This will take us to a new page where we can fill in importing the CSV and create the new records:
+
+![Page to import new records via csv](../../img/local_census/add-local-census-records-csv-en.png)
+
+## Features
+
+* Search by document_number: As local_census_records could contain a lot of records we have added a search feature to allow administrators to find existing records by document number.
```

Comment:
I think this sentence is missing a comma :thinking:. By the way, is local_census_records written with underscore because it's a table name? If so:

```suggestion
* Search by document number: As the `local_census_records` table could contain a lot of records, we have added a search feature to allow administrators to find existing records by document number.
```